src/App.tsx

```diff
@@ -59,7 +60,7 @@ setupIonicReact();
 const App: React.FC = () => {
   useEffect(() => {
     console.log("fetching...");
-    localStorage.setItem(LANGUAGE, LANG.ENGLISH);
+    localStorage.setItem(APP_LANG, LANG.ENGLISH);
```

Comment:
I think this LANGUAGE, we need it in cocos. @Sutara-Skanda  could you plz confirm